### PR TITLE
Add competition date announcement email

### PR DESCRIPTION
**Pull request by RealOrangeOne**
The original spec for this email included an update announcement, however I pulled that to prevent this email being blocked by the update.

### CODE REVIEW

File: SB2018/2018-02-27.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Hello all!
+
+I'm happy to confirm the dates for the competition this year! The competition will be held from *Saturday 21st April* until *Sunday 22nd April*. The competition will be held at the University of Southampton's Highfield campus, so please plan accommodation / transport accordingly. The exact start/end times are to be confirmed, but it'll start at some point in the morning of the 21st, and finish in the afternoon of the 22nd, no later than 5:00pm.
+
+This information will be reflected on our website soon!
+
+## Competition Progress
+
+You're now just over half way through this competition period. Hopefully you're making good progress with your robot, and are well on your way to having something competition ready!
```
Comment (Adimote) [inline SB2018/2018-02-27.md]: `s/just//`

Comment (Adimote) [reply SB2018/2018-02-27.md]: Random idea: Tweet #SourceBots with pictures of your robot!

File: SB2018/2018-02-27.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+There are just 2 remaining tech days between now and the competition. These are a great chance to spend a full day testing on your kit, and get direct support from us. The remaining tech days are:
+
+- 17th March
+- 14th April
+
+Both of these are likely to be held at Highfield campus, as with the last 2, however confirmation of this will come closer to the time.
+
+## Media Consent
+As with Student Robotics competition, media consent will be required to attend the competition. The forms for this will be sent through in the next week or two. These will need to be signed by each member of your team, as well as staff. Those under 18 will need a parent / guardian to sign on their behalf.
```
Comment (Adimote) [inline SB2018/2018-02-27.md]: Remove the 'As with the Student Robotics Competition', one thing we mentioned at kickstart was we shouldn't reference SR as much as we did.

**Comment by Adimote** [comment]
One last comment is we might want to change the subject line, the current one looks like it might be caught by spam detection, maybe "SourceBots Comp Date announced!".

**Comment by Adimote** [comment]
:shipit:

**Review by Adimote — APPROVED** [review]

File: SB2018/2018-02-27.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+You're now over half way through this competition period. Hopefully you're making good progress with your robot, and are well on your way to having something competition ready!
+
+## Tech Days
+
+There are just 2 remaining tech days between now and the competition. These are a great chance to spend a full day testing on your kit, and get direct support from us. The remaining tech days are:
+
+- 17th March
+- 14th April
+
+Both of these are likely to be held at Highfield campus, as with the last 2, however confirmation of this will come closer to the time.
```
Comment (trickeydan) [inline SB2018/2018-02-27.md]: Is it implied that Highfield Campus is at Southampton University? Not sure if we should assume this knowledge for teams who both didn't come to kickstart or any previous tech days.
EDIT: Ignore this, just seen that it was mentioned above

File: SB2018/2018-02-27.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+This information will be reflected on our website soon!
+
+## Competition Progress
+
+You're now over half way through this competition period. Hopefully you're making good progress with your robot, and are well on your way to having something competition ready!
+
+## Tech Days
+
+There are just 2 remaining tech days between now and the competition. These are a great chance to spend a full day testing on your kit, and get direct support from us. The remaining tech days are:
```
Comment (kierdavis) [inline SB2018/2018-02-27.md]: 2 -> two

Comment (kierdavis) [inline SB2018/2018-02-27.md]: 2 -> two

File: SB2018/2018-02-27.md (hunk 3 above)
Comment (kierdavis) [reply SB2018/2018-02-27.md]: 2 -> two (unrelated to above comment)

**Review by kierdavis — APPROVED** [review]
Looks fine to send as-is, though i've left a couple of pedantic comments as usual :)

**Pull request merged**